### ISSUE TITLE
image display distorted after windows update

### ISSUE THREAD

**Issue by jlader** [AntonyCorbett/OnlyM#393]
**Describe the bug**
image display is distorted.  Started after latest Win10 Update.  setting fading style to "None" corrects the issue

the 2 attached pictures show the same picture. with fading set to none, and fading set to crossfade.
![Fading None](https://user-images.githubusercontent.com/87709120/126337289-ee7652ae-688e-4e70-994a-16ee2346842f.jpg)
![fading crossfade](https://user-images.githubusercontent.com/87709120/126337306-ff16f72e-0c64-44a8-99e2-5ce1a6506828.jpg)

**Comment by AntonyCorbett** [comment]
@jlader Thanks for your report. Does it happen in Windows mode? Does it happen with landscape images? Does it happen using the latest pre-release of OnlyM?

**Comment by AntonyCorbett** [comment]
@jlader Have you tried rolling back your video driver. Windows Updates sometimes change the driver.

**Comment by jlader** [comment]
The pictures are from OnlyM in windowed mode.  with wide pictures, they 
distort up and down. (see attached).  I did not roll back the video 
driver, but I update to the latest Nvidia studio driver.  If you could 
send me a link to the pre-release, I can give it a try.

**Comment by EddyFernandez** [comment]
I have the same issue. I reported this to you via email but just wanted to add that I've tried reverting to 3 legacy video drivers and the outcome is just the same. My GPU is RTX 3080 Ti

**Comment by AntonyCorbett** [comment]
@jlader @EddyFernandez Does this issue remain? I am assuming it's caused by a Windows update interfering with the graphics driver.

**Comment by jlader** [comment]
Issue is still there.  Updated the Nvidia driver for the RTX2060 video card.  Perhaps it is an issue with RTX cards.  Short term fix - turned of the picture transitions.  Every now and then I check if anythings changed.

**Comment by AntonyCorbett** [comment]
@danielengels Do all monitors have the same scaling (in Windows Display Properties)?

**Comment by danielengels** [comment]
I find something strange.
When i use this picture for example
![80 327](https://user-images.githubusercontent.com/6145739/140385496-0c1a2a7f-2826-445a-a803-f6472ecb0f71.jpg)
It will be displayed like this:
![Unbenannt](https://user-images.githubusercontent.com/6145739/140385717-1b08256b-7d44-4544-b3b7-d43a58848667.JPG)
When I use magick and convert the image to a 16:9 (so also with 1920x1080) format and make the background black with this command magick "80 327.jpg" -resize 1280x720 -background black -gravity center -extent 1280x720 "80_327_3.jpg"
it is diplayed like expected.
It was also interessted to see, taht the image is "strange" caled in the preview
![Unbenannt JPG2](https://user-images.githubusercontent.com/6145739/140386092-05a5f967-f7a4-407a-b294-20050bd1bfaa.JPG)

Maybe this helps? Maybe we can check the dimension with some debug-information?
Thanks in advance
Daniel

**Comment by AntonyCorbett** [comment]
@danielengels it works fine here. I do notice that the image resolution in the metadata is 300 dpi whereas the original is 96. These figures are meaningless but does it indicate that the image has been processed in some way?

**Comment by danielengels** [comment]
The first pictures the original from jw.org.
The second is a screenshot with the windows snipping tool of the display of OnlyM.
The pictures processed with magick are working well. There is no strange aspect ratio and the pictures is not cutted. Here the procossed pictures with magick 
![80_327_3](https://user-images.githubusercontent.com/6145739/140397065-d6b78778-1e77-4c6f-a0e8-2e37c8af3e52.jpg)

**Comment by AntonyCorbett** [comment]
@danielengels To summarize:

1. You see distortion when using the original image (900x701)
2. The image is displayed correctly after you edit it to place it on a 1280x720 black background

Is that right? If so, perhaps you could experiment by processing the image in Magick but retaining the original pixel resolution.

It's difficult for me to do much here because I can't reproduce it (tested on 3 machines and a VM).

**Comment by jlader** [comment]
I have tried multiple image formats and settings without any change to effect.  Since the latest update, the image has not been distorted for me.  if image fading is turned on,  there is an odd flicked during the transition.  I have set the image fading style to none, which seems to be the source of the issue.  even in the settings menu, if fading is turned on, distortion occurs when accessing drop menus.  see attached and notice the distorted writing above the drop boxes.


![menu1](https://user-images.githubusercontent.com/87709120/140504433-bb7da536-5f82-47ed-bb6a-c6e2f092d890.jpg)

![menu2](https://user-images.githubusercontent.com/87709120/140504455-19dcbf0f-c9f1-456c-85fb-44471a94d298.jpg)

**Comment by jlader** [comment]
here is a short video showing flick during transition. 

https://user-images.githubusercontent.com/87709120/140505820-1b936512-7c61-42a3-ae19-e70f9f95bb23.mp4

**Comment by danielengels** [comment]
Yes, when i put the blendeffect off, there is also no problems displaying the pictures. This really works

**Comment by bradenpintar** [comment]
Just an update on this. I am breaking in a new laptop running Windows 11. The OnlyM version I am using is 2.0.0.12. I had the issue with the pix being stretched horizontally. I cycled through the fading styles and all of them stretched the image except for "None". I did not restart OmlyM while trying each of the options.

**Comment by jlader** [comment]
what are you system specs?

**Comment by bradenpintar** [comment]
Hello,

Hopefully, this helps...

[image: image.png]

[image: image.png]

If you need more information, please let me know.

Thanks.
--
Braden Pintar

**Comment by jlader** [comment]
pics didn't come through

**Comment by bradenpintar** [comment]
Let's try the old-fashioned way...

OS: Windows 11 Pro (Build 22000)
Make/Model: Dell G15 5511
Processor: 11th Gen Intel i7-11800H @ 2.30GHz, 2304 MHz 8 Cores, 16 Logical
Processors
BIOS: Dell 1.6.0 2021-09-28

Video 1
Intel UHD Graphics
Driver version: 30.0.100.9864
Resolution: 1920 x 1080 x 360 hertz

Video 2
NVIDIA GeForce RTX 3060 Laptop GPU
Driver version: 27.21.14.6685
Resolution: 1920 x 1080 x 60 hertz
--
Braden Pintar

**Comment by jlader** [comment]
thanks.  as advanced as the Nvidia RTX hardware/software is, I suspect that it is is creating issues.  I suspect it in other issues I am having.

**Comment by bradenpintar** [comment]
They kinda take over your video system.

I had in mind uninstalling all NVIDIA programs, deleting the card from the hardware list. Then, letting the system “find” the card again and loading just the driver.

To be fair, my laptop is a gaming machine. I do everything but on it, from database programming to 3D modeling. So…
